modules/stateful_config/variables.tf

```diff
@@ -22,6 +22,7 @@ variable "project_id" {
 variable "names" {
   description = "Instance names."
   type        = list(string)
+  default     = ["name1","name2","name3"]
```

Comment:
Seems that to have default names is not the best idea because people could use default names and don't pay attention to it and create resources with not the right names. 
I guess the variable was intentionally created without a default name.

Reply:
I could not plan the module out without specifying default values. maybe I am missing something here is there a way to work around that?

Reply:
@lpena-recurly 
How do you run a plan? 
The best way to test a module is to run a plan via `terragrunt.hcl` where you source the module

We can review this together if you want.